about/async/suggest.js

```diff
@@ -53,11 +54,50 @@ exports.create = function (api) {
       suggestion,
       {idle: true}
     )
-    suggestions = map(contacts, suggestion, {idle: true})
+
+    const suggestionsRecord = computed(contacts, contacts => {
+      var result = {}
+      contacts.forEach(contact => {
+        result[contact] = api.about.obs.names(contact)
+      })
+
+      return result
+    })
```

Comment:
oh sweet, thanks. 
We really need to start filling in some docs on mutant. I've thought about how it would be nice to be able to contribut things as I learn them to the docs, but I honestly haven't felt like I'm proficient enough to be able to come up with clear examples